pkgs/tools/graphics/vulkan-helper/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, vulkan-loader
+, addOpenGLRunpath
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vulkan-helper";
+  version = "2023-09-16";
+
+  src = fetchFromGitHub {
+    owner = "imLinguin";
+    repo = "vulkan-helper-rs";
+    rev = "d65b1a17a11ec20670c77d8da02e68d388ed0888";
+    hash = "sha256-usbYNalA0r09LXR6eV2e/T1eMNV4LnhzYLzPJQ6XNKQ=";
+  };
+
+  cargoSha256 = "sha256-s5QytuNhjZQhIDJtpeAW3J4op1t4nC+xD2i7Zf5mzfw=";
+
+  nativeBuildInputs = [
+    addOpenGLRunpath
+  ];
+
+  postFixup = ''
+    patchelf --add-rpath ${vulkan-loader}/lib $out/bin/vulkan-helper
+    addOpenGLRunpath $out/bin/vulkan-helper
```

Comment:
Are you sure this is necessary? The loader alone _should_ be enough.

Reply:
One of the program's subcommands crashes trying to `dlopen` an OpenGL library.

Reply:
Fun.